templates/toil-resource-group.yaml

```diff
@@ -0,0 +1,39 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  Resource group for toil cluster
+Parameters:
+  Department:
+    Description: 'The department tag for the resource group'
+    Type: String
+    AllowedPattern: '^\S*$'
+    ConstraintDescription: 'Must be string with no spaces'
+  Project:
+    Description: 'The project tag for the resource group'
+    Type: String
+    AllowedPattern: '^\S*$'
+    ConstraintDescription: 'Must be string with no spaces'
+  OwnerEmail:
+    Description: 'The email tag for the resource group'
+    Type: String
+    AllowedPattern: '^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$'
+    ConstraintDescription: 'Must be an acceptable email address syntax(i.e. joe.smith@sagebase.org)'
+  ClusterName:
+    Description: 'The name tag for the resource group'
+    Type: String
+Resources:
+  ResourceGroup:
+    Type: AWS::Inspector::ResourceGroup
+    Properties:
+      ResourceGroupTags:
+        -
+          Key: Department
+          Value: !Ref Department
+        -
+          Key: Project
+          Value: !Ref Project
+        -
+          Key: OwnerEmail
+          Value: !Ref OwnerEmail
+        -
+          Key: Name
```

Comment:
Actually i think cloudformation will automatically set the `Name` tag so it might be better to rename this to `ClusterName`?  or I guess we can merge and change it later as well.